Update notifications.md (#16)
test-IO/cirro-guides: src/pages/docs/features/notifications.md

### Notification Layouts
You can choose between two types of **Notification Layouts** to style your notification: Cirro (default) and Custom. If you use the Cirro (default) Layout, your message has a header with a logo, a body with the message and a footer with additional information. 

However, we recognize that Spaces are likely to reach a point where they do want control over every detail of their communication with users. In this case, you can create your own custom notification layout and attach your notification topics to it. This way you provide full layout plus content templates for your topics.

In case of an email notification, this means you will be responsible for the <html>, <head>, and <body> tags and everything there within.

#### Mandatory Placeholders
The one key caveat when you use custom layout is that you must now be sure to provide a placeholder in your email templates for the notice that Cirro legally must include in all emails to workers.
	
This notice will provide an unsubscribe link as well as additional information about the Cirro platform.

> You will be unable to create or update templates for a topic that uses custom layout unless that templates include the %{cirro_notice} placeholder somewhere.

Reviewer: inline code `%{cirro_notice}`

title: Notifications
description: Using the Cirro notification system.
---
## What are Cirro notifications?

> Communicating with your users is essential in order to maintain a well-functioning web application, thus a crowd-working service. In addition to the Cirro provided tools to make identity management, gig management, and invitation management simpler for you and your workers, we equip you with multiple means of communication.

Reviewer 2: replace term "workers" with users; you might need to be more specific at times when you refer to "users" in the context of Cirro and spaces - since spaces have different groups of users they entertain

Reviewer 3: `users` is actually correct here I believe. As I described in the Terminology section, `users` are everyone using your Space, `workers` are only users who get invited and perform work.
You definitely can use Notifications to contact any user, not only workers.

Reviewer 3: or did I misunderstand your comment?

Reviewer 3: Ah, I think I understand now, you were referring to the use of `worker` at the end of the paragraph. Then I agree :D

Author: Rewritten section based on the comments, it's currently pending review.

## Terminology
### Notifications
Before we clarify technical details, we need to understand notification-related terminology using our tree-planting example.

> All communication with users is managed via **Notifications**. A notification represents a particular message sent to a particular user at a particular point in time. The most common type of notification that a user will receive is the **"gig invitation"** notification that they receive each time when they match the criteria of a gig and have been invited to join using a variety of delivery mechanisms configurable based on project needs, utilising the Cirro worker interface and the user's email address. E.g.: A volunteer ('User' of the 'Space', once approved 'Worker') is invited to participate in a tree-planting event ('Gig') via an email or Cirro interface.

Reviewer: utilising the cirro worker interface?

Reviewer 3: Not `All communication` but `All offline communication` **can** be managed via Cirro Notifications - Spaces are not required to use Cirro Notifications if they want to implement their own system though.
Cirro Notifications cannot help with online/live communication like via Intercom or some other on-screen chat. It may be worth pointing that out.

> All communication with users is managed via **Notifications**. A notification represents a particular message sent to a particular user at a particular point in time. The most common type of notification that a user will receive is the **"gig invitation"** notification that they receive each time when they match the criteria of a gig and have been invited to join using a variety of delivery mechanisms configurable based on project needs, utilising the Cirro worker interface and the user's email address. E.g.: A volunteer ('User' of the 'Space', once approved 'Worker') is invited to participate in a tree-planting event ('Gig') via an email or Cirro interface.

Reviewer: ```suggestion
this is a suggestion
```

Reviewer 3: I'm confused, @thebluber you propose to replace the paragraph with `this is a suggestion`? ^^

> All communication with users is managed via **Notifications**. A notification represents a particular message sent to a particular user at a particular point in time. The most common type of notification that a user will receive is the **"gig invitation"** notification that they receive each time when they match the criteria of a gig and have been invited to join using a variety of delivery mechanisms configurable based on project needs, utilising the Cirro worker interface and the user's email address. E.g.: A volunteer ('User' of the 'Space', once approved 'Worker') is invited to participate in a tree-planting event ('Gig') via an email or Cirro interface.

Reviewer 2: ```suggestion
All communication with users is managed via **Notifications**. A notification represents a message sent to a user, in a specific way and at a particular point in time. A common type of notification that users receive is the **"gig invitation"** notification that they receive each time when they match the criteria of a gig and have been invited to join. A variety of delivery mechanisms are configurable based on the needs of your space.

E.g.: A volunteer is invited to participate in a tree-planting event via an email.```

Author: Rewritten section based on the comments, it's currently pending review.
